.gitignore

```diff
@@ -6,6 +6,7 @@ coverage.out
 *.iml
 *.swp
 *.log
+*.test
```

Comment:
It's better to keep '*.test' out of ignore list, we have some test files in `cmd/explaintest`.

Reply:
I have checked, this will not ignore the files in directory `cmd/explaintest`, just the file whose name ended with `.test`

Reply:
But we do have `.test` files: https://github.com/pingcap/tidb/tree/master/cmd/explaintest/t

Reply:
Oh, I see...

Reply:
Done ... renamed to *.test.bin